src/components/dialogs/parameters/load-flow-parameters.js

```diff
@@ -556,13 +454,12 @@ const SpecificLoadFlowParameters = ({
                 <FlatParameters
                     className={classes.parameterName}
                     paramsAsArray={specificParamsDescription}
-                    initValues={{
-                        ...defaultValues,
-                        ...lfParams?.specificParametersPerProvider?.[
-                            currentProvider
-                        ],
-                    }}
-                    showSeparator
+                    initValues={flatObject(
```

Comment:
Here you should only give `specificCurrentParams`

Reply:
fixed !